frontend/src/modules/fluenteditor/components/RichTranslationForm.js

```diff
@@ -328,42 +332,98 @@ export class RichTranslationFormBase extends React.Component<InternalProps> {
         />;
     }
 
-    renderItem(value: string, path: MessagePath, label: string, className: ?string) {
+    renderPluralExample(pluralForm: number) {
+        const examples = locale.getPluralExamples(this.props.locale);
```

Comment:
This is pretty excessive, in the case of Arabic, you're counting to 600.

I'd rather compute the examples in `renderElements`, and pass down the `examples | null` ?